pkg/sbom/v1/sbom_spdx_storage_format.go

```diff
@@ -74,40 +119,67 @@ func (sbom *SBOMData) StoreSBOM(sbomData any) error {
 	return nil
 }
 
+func (sbom *SBOMData) getSBOMData() (*spdxv1beta1.SBOMSPDXv2p3, error) {
+	file, err := os.Open(sbom.spdxDataPath)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	bytes, err := ioutil.ReadAll(file)
+	if err != nil {
+		return nil, err
+	}
+
+	spdxData := spdxv1beta1.SBOMSPDXv2p3{}
+	err = json.Unmarshal(bytes, &spdxData)
+	if err != nil {
+		return nil, err
+	}
+
+	return &spdxData, nil
+}
+
 func (sbom *SBOMData) FilterSBOM(sbomFileRelevantMap map[string]bool) error {
 	sbom.newRelevantData = false
+
+	spdxData, err := sbom.getSBOMData()
```

Comment:
I'm confused, you named the function `SBOMData`, then you name the local variable `spdxData`.
Is the `FilterSBOM` a converter from one format to another?